src/Designer/backend/src/Designer/TypedHttpClients/AzureDevOps/Models/QueueBuildRequest.cs

```diff
@@ -19,5 +19,11 @@ public class QueueBuildRequest
         /// </summary>
         [JsonPropertyName("parameters")]
         public string Parameters { get; set; }
+
+        /// <summary>
+        /// The source branch to build from (e.g., refs/heads/main)
+        /// </summary>
+        [JsonPropertyName("sourceBranch")]
+        public string SourceBranch { get; set; } = "refs/heads/feat/deploy-events";
```

Comment:
This should be removed before merge. Serves only to test pipeline from feature branch